### PR TITLE
Chapter 15 exercises

### PR DESCRIPTION
**Pull request by JPeace2**

### CODE REVIEW

File: chapter15/jxp/ch15ex2.pl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  when ( check_remainder( $_, 5) eq 0 ){$str .= "Bin "; continue}
+  when ( check_remainder( $_, 7) eq 0 ){$str .= "Sausage"}
+}
+if ( $str ){
+  say "$str\n";
+}else {
+  say "Not divisable by 3, 5 or 7";
+}
+
+sub check_remainder {
```
Comment (thatguy59) [inline chapter15/jxp/ch15ex2.pl:24]: Nice, this makes it a bit more readable.

File: chapter15/jxp/ch15ex2.pl [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+say "Please enter a number";
+chomp(my $num = <STDIN>);
+
+my $str;
+given($num){
+  when ( !looks_like_number($_) ){say "That is not a number, please try again";}
+  when ( check_remainder( $_, 3) eq 0 ){$str .= "Fizz "; continue}
+  when ( check_remainder( $_, 5) eq 0 ){$str .= "Bin "; continue}
+  when ( check_remainder( $_, 7) eq 0 ){$str .= "Sausage"}
```
Comment (thatguy59) [inline chapter15/jxp/ch15ex2.pl:16]: Shouldn't this use `==`?

File: chapter15/jxp/ch15ex3.pl [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+use feature "say";
+use 5.010001;
+
+foreach ( @ARGV ){
+  when ( ! -e $_ ){say "$_ does not exist"}
```
Comment (thatguy59) [inline chapter15/jxp/ch15ex3.pl:8]: This check could go in the when block

File: chapter15/jxp/ch15ex2.pl (hunk 2 above)
Comment (JPeace2) [inline chapter15/jxp/ch15ex2.pl:16]: Yes, well spotted!

File: chapter15/jxp/ch15ex3.pl (hunk 3 above)
Comment (qianliok) [inline chapter15/jxp/ch15ex3.pl:8]: I think it's already in `when` block?

**Pull request merged**